src/components/AudioAnalyzer/index.js

```diff
@@ -54,13 +61,26 @@ class AudioAnalyzer extends React.Component {
 
   render () {
     return (
-      <canvas ref={node => { this.canvas = node }} />
+      <div>
+        <canvas ref={node => { this.canvas = node }}
+          onMouseDown={this.props.onMouseDown || this.handleMouseDown}
+      />
+        {this.isOpen &&
+        <div style={{ position:'fixed' }}>
```

Comment:
We're using [`styled-components`](https://github.com/styled-components/styled-components) throughout the app, instead of inline styles. To give you an idea, it would look like this:

```javascript
import styled from 'styled-components'
import theme from '../../utils/theme'
...
const SettingsBox = styled.div`
   background: ${theme.bgColorDark1}; // use the theme to get a bg color
   position: absolute; // absolute makes more sense than fixed
`
...
<SettingsBox>
   <Control nodeId='audioNormalizeLevels' />
   <Control nodeId='audioLevelsFalloff' />
</SettingsBox>
```